.kuzzlerc.sample

```diff
@@ -50,18 +50,25 @@
     //      transformed in their Canonical Disjunctive Normal Form (CDNF).
     //      Set to 0 for no limit.
     // * subscriptionRooms
-    //      Maximum number of different subscription rooms 
+    //      Maximum number of different subscription rooms
     //      (i.e. different index+collection+filters subscription configurations)
     //      Depends primarily on available memory.
     //      If set to 0, an unlimited number of rooms can be created
+    // * subscriptionDocumentTTL
```

Comment:
@appacea Can you check my english please ?

Reply:
//      Maximum time (in seconds) a document will be kept in cache for real-time subscriptions.
//      This cache is used to notify subscribers when a document enters or leaves a scope after an update.
//      By default, subscriptions will be kept 72 hours.
//      /!\ Please note that keeping subscriptions over a long period of time can lead to a memory leak. /!\
//      If set to 0, the subscription will be kept in cache forever.

Reply:
TTLs set to extended periods of time won't lead to a memory leak fortunately, but most certainly to more memory consumed.
However, setting that property to 0 **will** (and not *can*) lead to a memleak **if and only if** documents enter a real-time subscription's scope, and never leave it again.

I think that it is very important to be precise here, because memory leaks are serious issues, and almost always the result of bugs, so we need to explain why a memory leak may occur.